docker/django/Dockerfile

```diff
@@ -70,7 +70,11 @@ RUN mkdir /var/log/courtlistener \
 
 WORKDIR /opt/courtlistener
 
+FROM python-base AS tailwind-reload
+CMD python /opt/courtlistener/manage.py tailwind install --no-input  && \
+    python /opt/courtlistener/manage.py tailwind start --no-input
 
+FROM python-base as web-dev
```

Comment:
We need to remove these lines. In https://github.com/freelawproject/courtlistener/issues/3134, we moved from a multi stage dockefile to a single image containing a helper script (`docker-entrypoint.sh`) that can be used to launch the different processes depending for different types of containers.